src/main/java/net/minecraftforge/items/wrapper/InvWrapper.java

```diff
@@ -0,0 +1,166 @@
+package net.minecraftforge.items.wrapper;
+
+import net.minecraft.inventory.IInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
+import net.minecraftforge.items.ItemHandlerHelper;
+
+public class InvWrapper implements IItemHandlerModifiable
+{
+    public final IInventory inv;
+
+    public InvWrapper(IInventory inv)
+    {
+        this.inv = inv;
+    }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
+
+        InvWrapper that = (InvWrapper) o;
+
+        return inv.equals(that.inv);
```

Comment:
Hm, I wonder if that is useful. Usually IInventories are not implementing equals and hashCode are they.